com.syntevo.SmartGit.json

```diff
@@ -0,0 +1,85 @@
+{
+  "app-id": "com.syntevo.SmartGit",
+  "command": "smartgit",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "separate-locales": false,
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--env=SMARTGIT_JAVA_HOME=/app/extra/smartgit/jre",
+    "--filesystem=host",
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "smartgit",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 apply_extra /app/bin/apply_extra",
```

Comment:
0755 is the default

Reply:
Noted, but this is more one of my "habits" – since it doesn't harm, I would rather leave it :wink: